net-dns/dnscap/dnscap-0_p20130814.ebuild

```diff
@@ -8,8 +8,8 @@ HOMEPAGE="https://dnscap.dns-oarc.net/"
 
 ## github commit tarball
 MY_GIT_COMMIT="727ed7d5e46625abc2c8d988689a300589e948b6"
-MY_P="verisign-${PN}-${MY_GIT_COMMIT:0:7}"
-SRC_URI="https://github.com/verisign/${PN}/tarball/${MY_GIT_COMMIT} -> ${PF}.tar.gz"
+MY_P="DNS-OARC-${PN}-${MY_GIT_COMMIT:0:7}"
+SRC_URI="https://github.com/DNS-OARC/${PN}/tarball/${MY_GIT_COMMIT} -> ${PF}.tar.gz"
```

Comment:
FTR, `${PF}` is incorrect here, it should really be just `${P}`.

Alternatively, you can avoid changing the tarball by replacing it with e.g. `${P/0_}` to retain the original filename, and having it fetched from Gentoo mirrors.

I guess either way is fine but if you prefer changing it, please use `${P}` instead of `${PF}` and make sure to diff both tarballs.

Reply:
The original file doen't corespond any more with the file from upstream (because of project owner / path change), so the ebuild don't work with the file from gentoo mirros since the changed checksum.
I have tried to modify as little as possible, since this is not my ebuild, 6 years old and hopefully get dropped in a near future. 

But ok, i will change ${PF} to ${P} ... please stand by